lib/rucio/tests/test_conveyor.py

```diff
@@ -757,6 +759,291 @@ def test_stager(rse_factory, did_factory, root_account, replica_client):
     assert replica['state'] == ReplicaState.AVAILABLE
 
 
+@skip_rse_tests_with_accounts
+@pytest.mark.noparallel(reason="runs submitter; poller and finisher")
+def test_transfer_to_mas_new_replica(rse_factory, did_factory, root_account):
+    """
+    Test qos: transfer from tape to disk
```

Comment:
This is actually from DISK to TAPE
I would add some more description, something like. Test QoS transfer from some RSE in the world to MAS